db/migrate/20210526091948_add_column_to_hall.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToHall < ActiveRecord::Migration[6.1]
+  def change
+    add_column :halls, :hall_number, :integer
```

Comment:
You can judge this column will be hall number from the name of the table. Simple
```
  add_column :halls, :number, :integer
```
would have sufficed.